src/coreclr/hosts/inc/coreclrhost.h

```diff
@@ -13,9 +13,24 @@
 // The prototype is useful for implicit linking against the dynamic coreclr
 // library and the pointer for explicit dynamic loading (dlopen, LoadLibrary)
 #define CORECLR_HOSTING_API(function, ...) \
-    extern "C" int function(__VA_ARGS__); \
-    typedef int (*function##_ptr)(__VA_ARGS__)
+    extern "C" int __stdcall function(__VA_ARGS__); \
```

Comment:
Does `__stdcall` work cross-plat?

Reply:
Sigh, it doesn't work on Unix ARM. On x86 / x64 Unix it is silently ignored.

Reply:
Fixed by making the __stdcall Win / x86 only.